packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -448,6 +448,10 @@ class AnimationController extends Animation<double>
   /// controller's ticker might get muted, in which case the animation
   /// controller's callbacks will no longer fire even though time is continuing
   /// to pass. See [Ticker.muted] and [TickerMode].
+  ///
+  /// If the animation was stopped (e.g. with [stop] or by setting a new [value]),
+  /// [isAnimating] will return `false` but the [status] will not change,
+  /// so the value of [AnimationStatus.isAnimating] might still be `true`.
```

Comment:
That's tricky. Would there be any benefit to documenting this difference on AnimationStatus.isAnimating as well?

Reply:
Ooh, hadn't thought of that!

I think the ideal place would be in the parent `Animation` class… I'll do that now.